projects/fsastorefrontlib/src/cms-components/my-account/policy/policies-chart/policies-chart.component.html

```diff
@@ -10,12 +10,12 @@
           'fscommon.paymentFrequency' | cxTranslate
         }}</span>
         <ng-select
+          [(ngModel)]="selectedFrequency.frequency"
```

Comment:
Add `ngModel` to track `selectedFrequency`, because when user select second item from dropdown and change language, it will reset dropdown and select first item. To prevent that behaviour I use ngModel to save last selected item from dropdown and display chart for that.